BookAPI/BookRepository.cs

```diff
@@ -116,10 +118,11 @@ public int AddNewBook(BookModel bookModel)
         {
             if (bookModel != null)
             {
-                UserRepository userRepository = new UserRepository();
+                //UserRepository userRepository = new UserRepository();
                 IsValidBookName(bookModel.BookName);
```

Comment:
Why validating individually again? Isn't ValidateBookModel already checking for everything?